dask/dataframe/io/parquet/core.py

```diff
@@ -710,10 +711,10 @@ def to_parquet(
             {"append": append, "compression": compression},
         )
     else:
-        dsk[name] = (lambda x: None, part_tasks)
+        dsk[(final_name, 0)] = (lambda x: None, part_tasks)
 
     graph = HighLevelGraph.from_collections(name, dsk, dependencies=[df])
-    out = Delayed(name, graph)
+    out = Scalar(graph, final_name, "")
```

Comment:
Looks like the docstring for to_parquet still says the return will be `Delayed` if `compute=False`